### PR TITLE
Added travis-ci

### PR DESCRIPTION
**Pull request by kavitamakariye**
@Hdonge  please review the PR.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,2 @@
 # simple-node-server
-Create simple node server
+Create simple node server [![Build Status](https://travis-ci.org/kavitamakariye/simple-node-server.svg?branch=master)](https://travis-ci.org/kavitamakariye/simple-node-server)
```
Comment (Hdonge) [inline README.md]: Markdown URL should be [Build Status](https://travis-ci.org/Hdonge/simple-node-server) of main upstream repo

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+language: node_js
+node_js:
+  - "8.10.0"
```
Comment (Hdonge) [inline .travis.yml]: New line

File: test/appTest.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+const assert = require('chai').assert;
+const index = require('../index');
+
+describe('test', () => {
+    it('Should return string', () => {
+        let result = 'ci with travis'
+        assert.equal(result,'ci with travis');
+    });
+});
```
Comment (Hdonge) [inline test/appTest.js]: New Line.

Test file name should match with the file which you are testing. It should be `test/index.test.js`
Also I see the test cases are not matching with file or functions in `index.js` file which should be tested. But I know the purpose of this repo is not the write the functional code and test it , but to understand git pr life cycle and integration with travis. Hence this is okay for now.

**Review by Hdonge — CHANGES_REQUESTED** [review]
Some minor review comments related to badge markdown url. Unit test related review can be ignored.

File: README.md (hunk 1 above)
Comment (Hdonge) [reply README.md]: image

Comment (kavitamakariye) [reply README.md]: Changed the Markdown to upstream repo

File: .travis.yml (hunk 2 above)
Comment (kavitamakariye) [reply .travis.yml]: Added a new line

File: test/appTest.js (hunk 3 above)
Comment (kavitamakariye) [reply test/appTest.js]: Changed the file name to index.test.js and added a new line.

**Comment by Hdonge** [comment]
@kavitamakariye Package.json file is showing some conflicts. Please fetch latest changes, resolved conflicts and then push this

**Review by Hdonge — APPROVED** [review]
Forgot to approve before merge. Code looks good. Approving now.

**Pull request merged**